client/post-editor/editor-sharing/accordion.jsx

```diff
@@ -46,12 +43,15 @@ const EditorSharingAccordion = React.createClass( {
 		}
 
 		const skipped = PostMetadata.publicizeSkipped( post );
```

Comment:
Another convenience of Lodash methods is that they automatically account for falsey values, so the condition at the top of this function could be simplified to:

```js
if ( ! isPublicizeEnabled ) {
	return;
}
```

...since `reduce` on an undefined `connections` or `includes` on an undefined `skipped` would be handled gracefully.